lib/hammer_cli_foreman_azure_rm/compute_resources/azure_rm.rb

```diff
@@ -1,6 +1,6 @@
 require 'hammer_cli_foreman/compute_resource/base'
 
-module HammerCLIForemanAzureRM
+module HammerCLIForemanAzureRm
   module ComputeResources
     class AzureRM < HammerCLIForeman::ComputeResources::Base
```

Comment:
@ofedoren Do you think the case change could affect existing `AzureRM` class that is inherited from the `Base` class? Mind changing it as `AzureRm` too? :)

Reply:
@apuntamb, well, the `AzureRM` constant is used within this plugin only, so changing this will affect the plugin itself only (I mean the name of this class can be any, e.g. we have in h-c-f `GCE`, `VMware`, etc)

But if you insist I can change it as well to be consistent in naming within the plugin :)

Reply:
@ofedoren Yeah just so that the consistency is maintained :)